src/Package/Impl/Repl/Commands/ReplCommandController.cs

```diff
@@ -152,10 +152,12 @@ public ReplCommandController(ITextView textView, ITextBuffer textBuffer)
             }
         }
 
-        private void HandleCancel(RCompletionController controller) {
-            Workflow.Operations.CancelAsync().DoNotWait();
-            // Post interrupt command which knows if it can interrupt R or not
-            VsAppShell.Current.PostCommand(RGuidList.RCmdSetGuid, RPackageCommandId.icmdInterruptR);
+        private void HandleCancel(RCompletionController controller, ITextView textView) {
```

Comment:
`TextView` is a field of the same class. Did you want to make this method static?